Examples/Python/python/acts/examples/simulation.py

```diff
@@ -625,16 +627,27 @@ def addGeant4(
 
     # Selector
     if postSelectParticles is not None:
-        postSelectedParticles = "particles_initial_selected"
+        particlesInitial = "particles_initial_selected"
         addParticleSelection(
             s,
-            preSelectParticles,
+            postSelectParticles,
             inputParticles=g4conf.outputParticlesInitial,
-            outputParticles=postSelectedParticles,
+            outputParticles=particlesInitial,
+        )
+
+        particlesFinal = "particles_final_selected"
```

Comment:
what is the use case for final particles selected?

Reply:
mainly that we write them out. In my case it makes quite a difference (over 100MB per event).

Reply:
that't a good point I didn't think of that :+1: